src/components/FullscreenContainer.tsx

```diff
@@ -25,12 +25,12 @@ const FullscreenContainer = ({ children }: Props): JSX.Element => (
       <Box
         alignItems="center"
         bgcolor="white"
-        border="1px solid #eaeaf7"
+        border="1px solid rgba(40, 40, 40, 0.4)"
```

Comment:
Same as above.

Reply:
Created a constant for this and the next within auth. These colors are not used outside of the AUTH tool, so no need to put them in the UI package.